File: generated-sources/kotlin-spring/mojang-authentication/src/main/kotlin/org/openapitools/model/ReducedUserData.kt
package org.openapitools.model

import java.util.Objects
import com.fasterxml.jackson.annotation.JsonProperty
import org.openapitools.model.GameProfileProperty
import javax.validation.Valid
import javax.validation.constraints.*

/**
 * 
 * @param id 
 * @param properties 
 */
data class ReducedUserData (

        @JsonProperty("id") val id: java.io.File? = null,

        @JsonProperty("properties") val properties: List<GameProfileProperty>? = null
) {

}

File: generated-sources/scala-gatling/mojang-authentication/src/gatling/scala/org/openapitools/client/model/ReducedUserData.scala

package org.openapitools.client.model

import java.io.File

case class ReducedUserData (
    _id: Option[File],
    _properties: Option[List[GameProfileProperty]]
)
object ReducedUserData {
    def toStringBody(var_id: Object, var_properties: Object) =
        s"""
        | {
        | "id":$var_id,"properties":$var_properties
        | }
        """.stripMargin
}
